web-components/src/components/read-more/index.tsx

```diff
@@ -59,7 +60,7 @@ const ReadMore: React.FC<ReadMoreProps> = ({
   const ReadButton = (): JSX.Element => (
     <OutlinedButton
       onClick={handleChange}
-      classes={{ root: classes.button, outlined: classes.buttonLabel }}
```

Comment:
MUI was throwing an error about the `outlined` classname. At first I thought this was because the `<OutlinedButton />` component can render as a `span` to handle images, but when I extracted that out it was still erroring (despite showing as a relevant field in the MUI docs). I believe combining the two achieved the desired look, but am having a hard time finding this in the mockups. How this looks:

<img width="881" alt="image" src="https://user-images.githubusercontent.com/9052511/156901947-b936845c-c310-4755-9854-9577cbad4483.png">